plugins/spamassassin.js

```diff
@@ -20,11 +24,11 @@ exports.load_spamassassin_ini = function () {
         max_size:     500000,
         old_headers_action: "rename",
         subject_prefix: "*** SPAM ***",
-        bypass_extra_headers: false,
+        add_headers: true,
     };
```

Comment:
you can remove add_headers from there, as declaring it as a boolean will insure it's initialized.